[2016-04-28 15:41:28.828,Pyro4.core,ERROR] cannot connect: receiving: connection lost: [Errno 104] Connection reset by peer
[2016-04-28 15:41:28.828,Pyro4.core,ERROR] cannot connect: receiving: connection lost: [Errno 104] Connection reset by peer
[2016-04-28 15:41:28.828,Pyro4.core,ERROR] cannot connect: receiving: connection lost: [Errno 104] Connection reset by peer
[2016-04-28 15:41:28.828,Pyro4.core,ERROR] problem getting metadata: cannot connect
[2016-04-28 15:41:28.828,Pyro4.core,ERROR] problem getting metadata: cannot connect
[2016-04-28 15:41:28.828,Pyro4.core,ERROR] problem getting metadata: cannot connect
[2016-04-28 15:41:28.828,Pyro4.core,ERROR] cannot connect: receiving: connection lost: [Errno 104] Connection reset by peer
[2016-04-28 15:41:28.828,Pyro4.core,ERROR] problem getting metadata: cannot connect
[2016-04-28 15:41:28.828,Pyro4.core,ERROR] cannot connect: receiving: connection lost: [Errno 104] Connection reset by peer
[2016-04-28 15:41:28.828,Pyro4.core,ERROR] cannot connect: receiving: connection lost: [Errno 104] Connection reset by peer
[2016-04-28 15:41:28.829,Pyro4.core,ERROR] problem getting metadata: cannot connect
[2016-04-28 15:41:28.829,Pyro4.core,ERROR] problem getting metadata: cannot connect
[2016-04-28 15:41:28.829,Pyro4.core,ERROR] cannot connect: receiving: connection lost: [Errno 104] Connection reset by peer
[2016-04-28 15:41:28.829,Pyro4.core,ERROR] problem getting metadata: cannot connect
